Project and locale switcher changes (#3293)
commercetools/merchant-center-application-kit: .changeset/six-needles-kiss.md

---
'@commercetools-frontend/application-shell': patch
---

> Removing the counters from locale and project swritchers. And adjusting their widths.

Reviewer: ```suggestion
Removing the counters from locale and project switcher in the app bar.
```
